.travis.yml

```diff
@@ -1,7 +1,6 @@
-language: java
+language: scala
```

Comment:
Nice for trying that out! I hope that this works as expected

Reply:
After this change, we don't need to download sbt ourself. Travis CI will include a sbt